### ISSUE TITLE
wine: The Elder Scrolls IV: Oblivion crashes on startup

### ISSUE THREAD

**Issue by SuperSamus** [NixOS/nixpkgs#215831]
### Describe the bug
When launching, using Wine, The Elder Scrolls IV: Oblivion (in my case, the Game of the Year Deluxe Edition from GOG, launched DRM free), the game freezes when it's supposed to load the main menu (that is, after the splash screens).

Wine version doesn't matter, like 32-bit only or WOW, I also tried Wine-GE from [nix-gaming](https://github.com/fufexan/nix-gaming), and tried with and without DXVK.

~Interestingly, if Wine is used from a FHS environment (like Bottles' or Heroic's), the game progresses a little further: it crashes only when it should show its main menu, and Wine doesn't log anything particular (nor show the window that says the program has crashed).~
Make sure to set PWD to the game's location.

I didn't test other similar games (like Fallout 3).

### Steps To Reproduce
1. Install Wine.
2. Launch `Oblivion.exe`
3. Freeze when it's supposed to load the main menu.

### Expected behavior
Game doesn't freeze.

### Metadata
```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.1.10-zen1, NixOS, 23.05 (Stoat), 23.05.20230209.6ccc4a5`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.2`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

```

**Comment by m911t** [comment]
Could this be related? I'm trying to play dragon's dogma: dark arisen, and the game closes right when the "press another button" screen comes up, and coincidentally that screen plays audio. I get a lot of messages about failed loading of .so files

**Comment by SuperSamus** [comment]
Never mind, duplicate of #132352.
If I use Wine 32 bit, the game launches, with working music. It crashes however if I don't launch it from the launcher. (And the launcher refuses to launch the game if you open it with from new prefix, that's why I didn't notice before. https://libreddit.spike.codes/r/linux_gaming/comments/pijhgt/fix_oblivion_launcher_has_all_options/)

**Comment by SuperSamus** [comment]
Though, I'm wondering, is it normal for the game to crash when launching `Oblivion.exe` (instead of `OblivionLauncher.exe`)?
Like, it's the same with Wine downloaded from https://github.com/Kron4ek/Wine-Builds/releases/ (with `steam-run`).
However, for some reason, it launches fine from Bottles, even when using `winePackages.unstable` from the system. It's not even the FHS environment, it launches fine even with `bottles-unwrapped`...

**Comment by L911P** [comment]
When i play with bottles or with steam's proton, Mod Organizer 2 just launches Oblivion.exe. It shouldn't crash. have you tried to disable music? "bMusicEnabled" in oblivion.ini

**Comment by SuperSamus** [comment]
Still crashes with `bMusicEnabled=0`.

**Comment by SuperSamus** [comment]
Reopening because I think it's supposed to launch successfully through `Oblivion.exe`.

By the way, I found out why there were differences in running the game between Bottles and regular terminal: Bottles was setting PWD to the executable location, and I wasn't doing that through the terminal. Stupid games that do this...
(The fun thing is that I was writing a whole issue about the differences between Bottles and terminal, because it affects more games than just Oblivion. Then I found out if was this, sigh...)

(#238991 didn't fix the issue, it only fixes `OblivionLauncher.exe`.)
